src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Windows/Native/Helpers.cs

```diff
@@ -73,7 +73,7 @@ public static byte[] ValueAsAscii(this Oid oid)
             return Encoding.ASCII.GetBytes(oid.Value);
         }
 
-        public unsafe delegate void DecodedObjectReceiver(void* pvDecodedObject);
+        public unsafe delegate void DecodedObjectReceiver(void* pvDecodedObject, int pvDecodedObjectLength);
```

Comment:
Since this is using Hungarian, the argument name should be `cbDecodedObjectLength`

Reply:
> the argument name should be cbDecodedObjectLength

Wouldn't it be `cbDecodedObject` in that case?  I just added the `Length` suffix to the `pvDecodedObject` name, but can switch to replacing the prefix.

Reply:
Yes, it can be `cbDecodedObject`. 

In Hungarian notation, the prefix is always based on either type or purpose - `void*` -> `pv`, `count of bytes` -> `cb`, etc.